web_development_101/project_html_css.md

```diff
@@ -94,6 +94,7 @@ There are a couple of ways to go about doing this, but the simplest is this:
 *Submit a link below to the github repo with your files in it by using a pull request.  See the section on [Contributing](http://github.com/TheOdinProject/curriculum/blob/master/contributing.md) for how.  Please include your partner's github handle somewhere in the description if you had one and they would like attribution.*
 
 * Add your solution below this line!
+* [Sanya Verma'solution](https://github.com/ErisedKT/google-homepage/) | [View in Browser] (https://erisedkt.github.io/google-homepage/)
```

Comment:
@ErisedKT There should not be a whitespace between `[View in Browser]` and `(https://erisedkt.github.io/google-homepage/)`

It should be `[View in Browser](https://erisedkt.github.io/google-homepage/)` 
Otherwise, the link won't be properly rendered. Try pasting your markdown into http://dillinger.io/ to see what I mean.